src/main/java/cloud/prefab/client/config/ConfigResolver.java

```diff
@@ -235,8 +385,6 @@ private String toS(Prefab.ConfigValue configValue) {
       return Double.toString(configValue.getDouble());
     } else if (configValue.getTypeCase() == Prefab.ConfigValue.TypeCase.SEGMENT) {
       return "Segment";
-    } else if (configValue.getTypeCase() == Prefab.ConfigValue.TypeCase.FEATURE_FLAG) {
```

Comment:
this toS method looks like a good case for a switch expression that can return values